ballerina/tests/test_entities_generated_types.bal

```diff
@@ -254,3 +259,49 @@ public type AllTypesIdRecordUpdate record {|
     string randomField?;
 |};
 
+
```

Comment:
```suggestion
```

Reply:
updated in [f7d8c93](https://github.com/ballerina-platform/module-ballerina-persist/pull/218/commits/f7d8c9364ffc8fa17d1d01c211a78e2af887b9bd)